applications/asset_tracker_v2/src/events/debug_module_event.c

```diff
@@ -25,7 +25,8 @@ static int log_event(const struct event_header *eh, char *buf,
 {
 	const struct debug_module_event *event = cast_debug_module_event(eh);
 
-	return snprintf(buf, buf_len, "%s", get_evt_type_str(event->type));
+	EVENT_MANAGER_LOG(eh, "%s", get_evt_type_str(event->type));
```

Comment:
In case of ble_event.c and hid_event.c you still use snprintf. In that case you actually snprintf to NULL, which should not happen.

Reply:
Keep in mind to add log_strdup there too.